Content.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Content({ children }) {
+  return (
+    <section>
+      { children }
```

Comment:
Maybe wrap that into a `<div>`. Like for example:

```suggestion
<div>
      { children }
</div>
```

Reply:
I agree.